README.md

```diff
@@ -45,12 +47,24 @@ It appears that this project could work with a **whole range of ASUS notebook mo
 
 ### Instructions
 
+#### Via GitPack
+
 This project uses [GitPack](https://github.com/dominiksalvet/gitpack). Global installation/update:
 
 ```sh
 sudo gitpack install github.com/dominiksalvet/asus-fan-control
 ```
 
+#### From AUR (for Arch-based distros)
+
+If you're using Arch Linux or Arch-based distribution, there's an [AUR package](https://aur.archlinux.org/packages/asus-fan-control) available, just [install](https://wiki.archlinux.org/index.php/Arch_User_Repository#Installing_packages) `asus-fan-control` from AUR. 
+
+The package doesn't automatically enable starting the service at boot (since Arch packages generally don't do that), to enable it at boot:
+
+```sh
+sudo systemctl enable asus-fan-control
+```
```

Comment:
BTW, is this really needed? Based on your comment, I understand that AUR packages generally don't enabled a service at the boot. But, I would like to have both GitPack and AUR installations consistent and hence having it enabled at boot would be required. Furthermore, I do not see any point of installing asus-fan-control on systemd-based system and not having it set up to run at boot...

So would it be possible to include changes that cause that after asus-fan-control installation it is set up to launch at boot, please? Or do you have any argument against it, except the provided one?

Reply:
It's not just AUR pacakges, it's Arch packages in general, they usually don't enable services unless it's some vital part of the system, so I'm keeping it consistent with the distribution.

Reply:
OK, I respect that. However, I am going to remaster *README.md* in the future. It is possible that this particular paragraph will be simplified but do not worry - no key information will be lost.